eclipse-scout-core/src/calendar/CalendarComponent.js

```diff
@@ -306,19 +313,27 @@ export default class CalendarComponent extends Widget {
     return dates.format(date, this.session.locale, pattern);
   }
 
-  _description() {
-    let descParts = [],
-      range = null,
+  _description(linkAllowed) {
+    let range = null,
       text = '',
```

Comment:
- Maybe this variable should be called htmlText or html as it contains html code and not just text?
- Or, create a description html component and at the end use `return $description.outerHtml`?
- Or, this method may return a Scout widget (Label) or directly the html component?